app/views/organizations/staff/staff/index.html.erb

```diff
@@ -19,7 +19,7 @@
           </div>
         </div>
       </div>
-      <%= link_to "Invite Staff", new_user_invitation_path, class: "btn btn-primary" %><br>
+      <%= link_to "Invite Staff", new_staff_staff_invitation_path, class: "btn btn-primary" %><br>
```

Comment:
Could we change this to `/staff/invitations/new` or will that be annoying to refactor?

Reply:
Hmm yes it is a bit of a mouthful. I would vote we keep the `StaffInvitationsController` name, but just amend the path in the routes to be new_staff_invitation_path